src/pages-and-resources/discussions/app-list/AppList.jsx

```diff
@@ -21,14 +21,18 @@ function AppList({ intl }) {
   const apps = useModels('apps', appIds);
   const features = useModels('features', featureIds);
 
+  useEffect(() => {
+    // If selectedAppId is not set, use activeAppId
+    if (!selectedAppId) {
+      dispatch(selectApp({ appId: activeAppId }));
+    }
+  }, [selectedAppId, activeAppId]);
```

Comment:
if you can add a little description / different between the `selectedAppId, activeAppId` that would be great. 

What is `selectedAppId` what it is referring to? How it is different from `activeAppId`?

Reply:
Done!